src/k8s/pkg/k8sd/setup/kube_apiserver_test.go

```diff
@@ -132,6 +132,19 @@ func TestKubeAPIServer(t *testing.T) {
 		g.Expect(len(args)).To(Equal(len(tests)))
 	})
 
+	t.Run("ArgsDualstack", func(t *testing.T) {
+		g := NewWithT(t)
+
+		s := mustSetupSnapAndDirectories(t, setKubeAPIServerMock)
+
+		// Setup without proxy to simplify argument list
+		g.Expect(setup.KubeAPIServer(s, "10.0.0.0/24,fdXY:IJKL:MNOP:15::/64", "https://auth-webhook.url", false, types.Datastore{Type: utils.Pointer("external"), ExternalServers: utils.Pointer([]string{"datastoreurl1", "datastoreurl2"})}, "Node,RBAC")).To(BeNil())
```

Comment:
let's make sure this is an actual IPv6 CIDR :)

```suggestion
		g.Expect(setup.KubeAPIServer(s, "10.0.0.0/24,fd01::/64", "https://auth-webhook.url", false, types.Datastore{Type: utils.Pointer("external"), ExternalServers: utils.Pointer([]string{"datastoreurl1", "datastoreurl2"})}, "Node,RBAC")).To(BeNil())
```